catalog/ui/search-ui/standard/src/main/webapp/js/HandlebarsHelpers.js

```diff
@@ -334,6 +334,32 @@ define([
                     return encodeURIComponent(str);
                 }
                 return str;
+            },
```

Comment:
@andrewkfiedler @stustison @pklinef are we good with calling out "MapActions" this way?